Mathlib/CategoryTheory/Monoidal/Cartesian/Mod_.lean

```diff
@@ -0,0 +1,26 @@
+/-
+Copyright (c) 2025 Paul Lezeau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Paul Lezeau
+-/
+import Mathlib.CategoryTheory.ChosenFiniteProducts
+import Mathlib.CategoryTheory.Monoidal.Mod_
+
+/-!
+# Additional results about module objects in cartesian monoidal categories
+-/
+
+open CategoryTheory MonoidalCategory ChosenFiniteProducts
+
+universe v u
+variable {C : Type u} [Category.{v} C] [ChosenFiniteProducts C]
+
+/-- Every object is a module over a monoid object via the trivial action. -/
+@[simps]
+def Mod_.trivialAction (M : Mon_ C) (X : C) : Mod_ M where
+  X := X
+  act := snd M.X X
+
+/-- Every object is a module over a monoid object via the trivial action. -/
+@[reducible] def Mon_Class.trivialAction (M : C) [Mon_Class M] (X : C) : Mod_Class M X where
```

Comment:
```suggestion
@[reducible] def Mod_Class.trivialAction (M : C) [Mon_Class M] (X : C) : Mod_Class M X where
```
Is this a typo or was this intended?

Reply:
I think that's intended

Reply:
I must say `Mod_Class.trivialAction` seems a bit better because it allows anonymous dot notation